client/src/clients/utils.ts

```diff
@@ -94,7 +94,7 @@ function getClientURL(
   clientType: ClientType = "public",
   transportType: "http" | "webSocket" = "http"
 ): string | undefined {
-  if (chain?.id !== 43_113 && chain?.id !== 43_114) {
+  if (chain?.id !== 43_112 && chain?.id !== 43_113 && chain?.id !== 43_114) {
```

Comment:
nit: now that I am thinking about it instead of increasing the conditions we can maintain a list of chainIDs keep it in a const file and remove the 3 conditions here.